extension.js

```diff
@@ -458,6 +472,40 @@ function downloadScript(receivedMsg) {
     }
 }
 
+function gslDateCheck(context) {
+    let editor = vscode.window.activeTextEditor;
+    if (!editor) {
+        return vscode.window.showErrorMessage('You must have a script open before you can check its date.');
+    }
+    let doc = editor.document;
+    if (!doc) {
+        return vscode.window.showErrorMessage('You must have a script open before you can check its date.');
+    }
+    let scriptNum = /([1-9]\d+)/.exec(doc.fileName)[1];
+    if (!/^\d{1,5}$/.test(scriptNum)) {
+        return vscode.window.showErrorMessage('Unable to parse script # from file name.');
+    }
+    gslEditor.scriptNum = scriptNum;
+    gslEditor.dateCheck = 1;
```

Comment:
Add `vscode.window.setStatusBarMessage('Checking last modified date of script ' + scriptNum + '...', 5000);` on the following line.